setup.cfg

```diff
@@ -52,6 +52,7 @@ filterwarnings =
     error
     # Suppress deprecation warnings in other packages
     ignore:lib2to3 package is deprecated::scspell
+    ignore:pkg_resources is deprecated as an API::flake8_import_order
```

Comment:
Spirit of the stairwell review comment: Do we have a tracking issue for the pkg_resources usage in colcon overall which we could add a comment to for this suppression?

Reply:
All of these suppressions are for problems in other packages, so there's nothing we can do in colcon to fix them. The only reason they need to be called out like this is that we're setting `filterwarnings = error` to make noise about problems in colcon itself. Unfortunately, we can't use wildcards in this filter, because `filterwarnings = error::colcon.*` is what we actually want.